src/components/Pips.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import "./Pips.css";
+
+export const Pips: React.FC<{ numThrown: number }> = ({ numThrown }) => (
+  <div className="pips">
+    {numThrown > 0 ? <div className="filled" /> : <div className="empty" />}
```

Comment:
this is easier to read

```suggestion
    <div className="{numThrown > 0 ? "filled" : "empty" }" />
```

etc..

Reply:
oh yeah, i guess that was a pretty naive solution, wasn't it 🫤